pages/login.js

```diff
@@ -0,0 +1,58 @@
+import Head from 'next/head';
+import { useState } from 'react';
+import styles from '../styles/Home.module.css';
+
+export default function Login() {
+  const [errorMessage, setErrorMessage] = useState('');
+
+  function onSubmit(event) {
+    event.preventDefault()
+
+    // Get form data
+    const formData = new FormData(event.target)
+
+    let username = formData.get("username")
+    let password = formData.get("password")
+
+    // Check if user input username and password
+    if (username.length == 0 || password.length == 0) {
```

Comment:
Strict equality check because we know the length will be an integer
```suggestion
    if (username.length === 0 || password.length === 0) {
```